packages/arb-token-bridge-ui/src/components/TransactionHistory/TransactionsTableRowAction.tsx

```diff
@@ -81,19 +100,53 @@ export function TransactionsTableRowAction({
     }
   }
 
-  if (tx.status === 'Unconfirmed') {
+  if (isDepositReadyToRedeem(tx)) {
+    // Failed retryable
     return (
       <Tooltip
+        show={isRedeemButtonDisabled}
         wrapperClassName=""
-        content={<span>Funds aren&apos;t ready to claim yet</span>}
+        content={
+          <span>
+            Please connect to the L2 network to re-execute your deposit. You
```

Comment:
can be L3 though? can we replace `L2 network` with the chain name?

Reply:
updated